shared/constants.js

```diff
@@ -30,45 +30,72 @@ var VERSION = '6.0.0';
     version: VERSION,
 
-    tools: {
-        gitMinVersion: '2.13',
-        npmMinVersion: '3.10',
-        podMinVersion: '1.2'
-    },
-    
-    cordova: {
-        pluginRepoUri: 'https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin#dev',    // dev
-        //pluginRepoUri: 'https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin#v' + VERSION, // GA
-        minimumCliVersion: '7.0.0',
-        platformVersion: {
-            ios: '4.4.0',
-            android: '6.2.3'
-        }
-    },
+    templatesRepoUri: 'https://github.com/forcedotcom/SalesforceMobileSDK-Templates#dev',    // dev
+    //templatesRepoUri: 'https://github.com/forcedotcom/SalesforceMobileSDK-Templates#v' + VERSION, // GA
 
-    appTypes: {
-        ios: ['native', 'native_swift', 'react_native', 'hybrid_local', 'hybrid_remote'],
-        android: ['native', 'native_kotlin', 'react_native', 'hybrid_local', 'hybrid_remote']
-    },
-
-    templates: {
-        repoUri: 'https://github.com/forcedotcom/SalesforceMobileSDK-Templates#dev',    // dev
-        //repoUri: 'https://github.com/forcedotcom/SalesforceMobileSDK-Templates#v' + VERSION, // GA
-        appTypesToPath: {
-            ios: {
+    forceclis: {
+        forceios: {
+            name: 'forceios',
+            dir: 'ios',
+            platforms: ['ios'],
+            toolNames: ['git', 'npm', 'pod'],
+            ide: 'XCode',
+            appTypes: ['native', 'native_swift', 'react_native'],
+            appTypesToPath: {
                 'native': 'iOSNativeTemplate',
                 'native_swift': 'iOSNativeSwiftTemplate',
-                'react_native': 'ReactNativeTemplate',
-                'hybrid_local': 'HybridLocalTemplate',
-                'hybrid_remote': 'HybridRemoteTemplate'
-            },
-            android: {
+                'react_native': 'ReactNativeTemplate'
+            }
+        },
+        forcedroid: {
+            name: 'forcedroid',
+            dir: 'android',
+            platforms: ['android'],
+            toolNames: ['git', 'npm'],
+            ide: 'Android Studio',
+            appTypes: ['native', 'native_kotlin', 'react_native'],
+            appTypesToPath: {
                 'native': 'AndroidNativeTemplate',
                 'native_kotlin': 'AndroidNativeKotlinTemplate',
-                'react_native': 'ReactNativeTemplate',
+                'react_native': 'ReactNativeTemplate'
+            }
+        },
+        forcehybrid: {
+            name: 'forcehybrid',
+            dir: 'hybrid',
+            platforms: ['ios', 'android'],
+            toolNames: ['git', 'npm', 'cordova'],
+            ide: 'XCode and/or Android Studio',
+            appTypes: ['hybrid_local', 'hybrid_remote'],
+            appTypesToPath: {
                 'hybrid_local': 'HybridLocalTemplate',
                 'hybrid_remote': 'HybridRemoteTemplate'
             }
         }
+    },
+
+    tools: {
+        git: {
+            checkCmd: 'git --version',
```

Comment:
The command to run to get the current version of a tool is now in constants.js instead of being in the code.